java-spiffe-core/src/main/java/io/spiffe/workloadapi/retry/BackoffPolicy.java

```diff
@@ -0,0 +1,83 @@
+package io.spiffe.workloadapi.retry;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+import lombok.val;
+
+import java.time.Duration;
+import java.util.function.UnaryOperator;
+
+/**
+ * Represents a backoff policy for performing retries using exponential increasing delays.
+ */
+@Data
+public class BackoffPolicy {
```

Comment:
Can we call this something like `ExponentialBackoffPolicy` to be explicit about what kind of policy is implemented here?

Reply:
Renamed it to `ExponentialBackoffPolicy` and change the option to configure the policy, instead of allowing to set the unary operator to calculate the delay, now it is allowed to configure the multiplier, to keep it consistent with the name, otherwise the user could set a function that be not exponential.